plugins/main/public/utils/applications.ts

```diff
@@ -718,7 +718,7 @@ const wazuhPluginLogs = {
   category: 'management',
   id: 'wazuh-plugin-logs',
```

Comment:
**nitpick**: rename to `app-logs`.

Reply:
done [b1a4838](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6086/commits/b1a483869e88f2ab865dfddf90829d9628153202)